docs/getting-started.md

```diff
@@ -4,7 +4,7 @@ title: Getting Started
 ---

```

Comment:
hier moeten we het nog over hebben. Gaan we binnen de master documentatie opnemen voor 2 versies? Of moet iemand die nog de v1.3 nodig heeft daarvoor naar de release/tag 1.3?

In het ene geval moet dit dan hernoemd worden naar Getting-started-v1.md. 
In het tweede geval moet dit document worden verwijderd (en kan uit Getting-started-v2 de versie "-v2" worden verwijderd).